cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/common/ClusterProperty.java

```diff
@@ -16,7 +16,8 @@
   MEAN_CPU("meanCpu"),
   MEAN_DISK("meanDisk"),
   MEAN_NW_IN("meanNwIn"),
-  MEAN_NW_OUT("meanNwOut");
+  MEAN_NW_OUT("meanNwOut"),
+  POPULATE_DISKS("populateDisks");
```

Comment:
`POPULATE_DISKS("populateDisks")` -> this name is not very clear, can we either rename or add a comment to explain what it is?